### ISSUE TITLE
NestedClassesVisibility doesn't detect violation for "object"

### ISSUE THREAD

**Issue by MyDogTom** [arturbosch/detekt#814]
version: 1.0.0.R6-4

```kotlin
internal sealed class Alphabet {
    object A : Alphabet()
    data class B(val throwable: Throwable) : Alphabet()
}
```

Rule detects violation for `B`, but doesn't detect violation for `A`

**Comment by schalkms** [comment]
Thanks for the feedback!
